### PR TITLE
NEWS.md: update for 0.7.3

### PR DESCRIPTION
**Pull request by johanmalm**
Review(s) appreciated.

Intend to merge on Friday.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -51,6 +65,34 @@ The format is based on [Keep a Changelog]
 
 ### Fixed
 
+- When looking for menu.xml, go through all paths rather than just giving up
+  if there is no menu.xml in the first path searched? This makes it consistent
+  with how other config/theme files are handled. PR #1971
```
Comment (Consolatis) [inline NEWS.md:70]: First sentence is a question

**Comment by jp7677** [comment]
```
- Add tablet-v2 pad support. Written-by: @jp7677 PR #1882
- Add partial support for `tablet-v2-manager`. Written-by: @jp7677 PR #1678
```

May be combine those two into

```
- Add support for `tablet-v2-manager`. Written-by: @jp7677 PR #1678 #1882
```

File: NEWS.md [hunk 2]
```diff
@@ -51,6 +65,34 @@ The format is based on [Keep a Changelog]
...
+- Fix memory leaks in theme.c and menu.c. PR #1971
+- Fix session-lock bugs related to keyboard focus. PR #1952
+  - Clear focused surface on lock
+  - Restore focused view on unlock
+  - Never allow multiple session locks
```
Comment (tokyo4j) [inline NEWS.md:75]: The last one is what I introduced and fixed in the same release cycle so I think can be omitted.

File: NEWS.md (hunk 1 above)
Comment (johanmalm) [reply NEWS.md]: Done.

File: NEWS.md (hunk 2 above)
Comment (johanmalm) [reply NEWS.md]: Done.

Assume you meant to just remove "Never allow multiple session locks" rather than all four lines relating to #1952

**Pull request merged**